### PR TITLE
[WIP] Add support for Environment Variables for Deployment

### LINKED ISSUE #473: [Backend] Add environment variables to deploy information
As a user, I want to be able to set environment variables for  *MicoServices* to configure them.
They have to be set during the deployment.

### PR DESCRIPTION
**Pull request by JanVetter**
Resolves #473 .

### CODE REVIEW

File: mico-core/src/main/java/io/github/ust/mico/core/model/MicoEnvironmentVariable.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*
```
Comment (JanVetter) [inline mico-core/src/main/java/io/github/ust/mico/core/model/MicoEnvironmentVariable.java]: There are two classes (`MicoEnvironmentVariable`, `MicoLabel`) consisting only of a Key-Value pair. Can't we just use one of them for both?

Comment (davidkopp) [reply mico-core/src/main/java/io/github/ust/mico/core/model/MicoEnvironmentVariable.java]: No. They are different. For example the `MicoEnvironmentVariable` has the properties `name` and `value` and the `MicoLabel` has the properties `key` and `value`. Furthermore they require different annotation e.g. for validation.

**Comment by davidkopp** [comment]
Waiting for fix of updating deployment information in #556.

**Comment by davidkopp** [comment]
Environment variables are not stored in Neo4j, because list are not supported as properties on `RelationsshipEntities`. @JakobBenz will fix this...

**Pull request closed without merging**